.github/workflows/format_repositories.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       -
```

Comment:
Nit: you don't need a blank link when starting a new object in an array in yaml. I think this would look better:
```yaml
    - run: dotnet build csharpier/Src/CSharpier/CSharpier.csproj -c release
```